documentation/publish.md

```diff
@@ -64,8 +64,7 @@ Once your extension is signed, it is subject to review by Mozilla at any time. T
 There are four ways you can distribute your extension. They are:
 
 - Use [addons.mozilla.org](https://developer.mozilla.org/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on) (most popular, broadest audience)
-- Provide a download location for the signed extension file for users to self-install or [sideload](/documentation/publish/distribute-sideloading/) into Firefox.
-- Include the extension installation as part of the installation of a [desktop application](/documentation/publish/distribute-for-desktop-apps/).
+- Provide a download location for the signed extension file for users to self-install into Firefox.
```

Comment:
Sorry @apsychogirl - can you link "download extension" to `distribute-sideloading/`? My fault for not including the link in the issue!